setup.cfg

```diff
@@ -41,6 +41,8 @@ packages = find:
 test =
     cython-test-exception-raiser ~= 1.0
     PyHamcrest >= 1.9.0
+    pydoctor >= 20.7.0
```

Comment:
Thanks for the great start.


The updates to `test` are not quite ok.

In Twisted we are running tests without dependencies to make sure that twisted can handle soft dependencies.

We should not add more deps to this... and the long term plan is to also remove pyhamcrest or skip hamcrest test when pyhamcrest is not installed.

Yes... this did the trick, but I think that we should add a dev_docs extra requirements.

I will try to see if I can make it right  :)